src/blocks/container.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  padding: 50px 40px;
+  flex-wrap: wrap;
+  width: 1024px;
```

Comment:
don't hardcode container width, it should be calculated according to the card width and amount of the cards